lib/ansible/modules/crypto/luks_device.py

```diff
@@ -197,6 +234,30 @@ def __init__(self, module):
     def _run_command(self, command):
         return self._module.run_command(command)
 
+    def get_device_by_uuid(self, uuid):
+        ''' Returns the device that holds UUID passed by user
+        '''
+        self._blkid_bin = self._module.get_bin_path('blkid', True)
+        uuid = self._module.params.get('uuid')
+        if uuid is None:
+            return None
+        result = self._run_command([self._blkid_bin, '--uid', uuid])
```

Comment:
```suggestion
        result = self._run_command([self._blkid_bin, '--uuid', uuid])
```
According to `man blkid` on my system, it is `--uuid` and not `--uid`.

Reply:
> > > It looks like you pushed the wrong version, the current version undoes a lot of things from the previous pushes.
> > 
> > 
> > My bad, two computers, a lot of rebases, I revised last versions and seems its OK now.
> 
> That's why it's best not to rebase, but to add new commits :)

I always tought that create a lot of commits was not a good idea, but some of the rebases made here was not cool also, lesson is: dont write code with a late pregnant wife and with your kid of 2 wanting to play, wait for a calm day and write these on the right way (thanks God for all that). But its always the coder not the family behind him :D

sorry for made you waste your time, hope now, everything is OK.